en/docs/learn/analytics/managing-dashboard-permissions.md

```diff
@@ -48,3 +145,30 @@ The following table illustrates the default mapping between the permission level
 | Internal/publisher  | Not allowed               | Not allowed            | Viewer            | Not allowed    | Not allowed    |
 | Internal/subscriber | Viewer                    | Not allowed            | Not allowed       | Not allowed    | Not allowed    |
 
+
+Further, you can add custom scopes for WSO2 API Manager Analytics. In order to do that, you have to add the custom scope and role mapping to `RESTAPIScopes` value of `APIM_HOME/repository/resources/tenant-conf.json` file and configure that in `ANALYTICS_HOME/conf/dashboard/deployment.yaml`.
+
+For example if you need to create a custom scope `apim_analytics:reports:view`,  and map with `Internal/analytics` and `Internal/creator` role, then you have to add  the following entry to `RESTAPIScopes` value.
+
+``` toml
```

Comment:
in the sample,  don't see where "RESTAPIScopes" is.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/1697/commits/ab12d89bb02874fd1eccd6b48d0a387a8ccd7ffa